src/models/constants/JobTypes.js

```diff
@@ -1,6 +1,6 @@
 const JobTypes = Object.freeze([
-    "FULL_TIME",
-    "PART_TIME",
+    "FULL-TIME",
+    "PART-TIME",
```

Comment:
It might make sense to move this to an external package, to promote code reuse inside nijobs between FE and BE, as discussed.

Reply:
hehe, read my thoughts, let's do this in another PR though

Reply:
Created #103 to track this

Reply:
Thanks, that's a great idea!